lib/engine/ownable.rb

```diff
@@ -14,6 +14,10 @@ def player
       owner.is_a?(Player) ? owner : owner&.player
     end
 
+    def corporation
+      is_a?(Corporation) ? self : owner&.corporation
```

Comment:
corporation?

Reply:
Should the `is_a?(Player)` calls here be replaced with `.player?`? This has the odd side effect of ownable things needing to implement basically everything on `Entity`. Should `Ownable` have `include Entity`?

Anyway with `corporation?` or with doing all the changes I mentioned, I'm getting a bunch of errors, will come back to this one later.

Reply:
Got this working, can't be 100% sure but the first time around I think I was missing some `&`s when `owner` might be `nil`